<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="WIN-81NG0PGA72A" ignored="0" name="Test" tests="1" failures="1" timestamp="23 Oct 2018 07:58:51 GMT" time="125.879" errors="0">
  <testcase name="myAccountSummary" time="62.412" classname="Testcases.MyAccountScripts">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Testcases.MyAccountScripts.myAccountSummary(MyAccountScripts.java:31)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- myAccountSummary -->
</testsuite> <!-- Test -->
